Classes/Util.php

```diff
@@ -252,44 +252,60 @@ public static function getConfigurationFromPageId(
         $language = 0,
         $useCache = true
     ) {
-        static $configurationsByCacheKey = array();
+        static $configurationObjectCache = [];
         $cacheId = md5($pageId . '|' . $path . '|' . $language);
-
-        if (isset($configurationsByCacheKey[$cacheId])) {
-            return $configurationsByCacheKey[$cacheId];
+        if (isset($configurationObjectCache[$cacheId])) {
+            return $configurationObjectCache[$cacheId];
         }
 
         // If we're on UID 0, we cannot retrieve a configuration currently.
         // getRootline() below throws an exception (since #typo3-60 )
         // as UID 0 cannot have any parent rootline by design.
         if ($pageId == 0) {
-            return self::buildTypoScriptConfigurationFromArray(array(), $pageId, $language, $path);
+            return self::buildTypoScriptConfigurationFromArray([], $pageId, $language, $path);
```

Comment:
Can we `return []` instead? Alternatively, if this method can be expensive, should we allow this to become cached? If so, we have to move the "useCache" decision up above this point and not return from this condition but assign $configurationArray instead (or cache-assign-and-return as is done below).